libcxx/modules/std.compat/cstdlib.inc

```diff
@@ -16,10 +16,10 @@ export {
   // [support.start.term], start and termination
   using ::_Exit;
   using ::abort;
-  using ::at_quick_exit;
+  using ::at_quick_exit _LIBCPP_USING_IF_EXISTS;
```

Comment:
@mordante We should really be using it for all C declarations we import. That would be required to work with libraries that e.g. don't provide support for localization and other similar things.

Reply:
@ldionne we indeed probably should, I really hope we can start to make progress adding more CI testing for modules. For now I prefer to keep it so this lands in LLVM 18.